CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Release Notes
 
+## Version 1.29.0 - 2023-01-26
+
+Features:
+* Graduate QEMU driver from experimental to GA [#15556](https://github.com/kubernetes/minikube/pull/15556)
+* Implement --static-ip flag (Docker only) [#15553](https://github.com/kubernetes/minikube/pull/15553)
+* Implement Kicbase/ISO version validation when starting existing clusters [#15235](https://github.com/kubernetes/minikube/pull/15235)
+* GCP-Auth addon automatically integrates with newly created namespaces [#15403](https://github.com/kubernetes/minikube/pull/15403)
```

Comment:
add a colon after the name
GCP Auth-Addon: automatically attach credentials to newly created namesapces